my_game.py

```diff
@@ -416,9 +395,9 @@ def on_draw(self):
             arcade.color.WHITE  # Color of text
         )
         arcade.draw_text(
-            "LIVES: {}".format(self.player_sprite.lives),  # Text to show
+            "LIVES: {}".format(self.player_lives),  # Text to show
             10,  # X position
```

Comment:
Don't make this change. We want the _Player_ to keep track of it's own lives.

Reply:
But here is the problem with that. Every time I use self.setup() it makes it lives to 3 and when you die you lose one. It works first time but after that it will keep losing 1 life. Making it 2 every time you touch an asteroid.

Reply:
Never mind fixed it